### ISSUE TITLE
internal/gocdk: cloudrun launch fails (due to invalid credentials)

### ISSUE THREAD

**Issue by vangent** [google/go-cloud#2335]
The error message is lacking:

```
$ gocdk launch gcp
Error: gocdk launch: cloud run launch: docker push: exit status 1
```

When I modified the [code](https://github.com/google/go-cloud/blob/master/internal/cmd/gocdk/internal/launcher/cloudrun.go#L54) to send the output from `docker` to `os.Stderr` (there is a TODO to send the output somewhere other than `ioutil.Discard`), I get this:

```
The push refers to repository [gcr.io/go-cloud-test-216917/foo]
2cc15c479ba8: Preparing 
87c747af6dc3: Preparing 
01092e5921c5: Preparing 
unauthorized: You don't have the needed permissions to perform this operation, and you may have invalid credentials. To authenticate your request, follow the steps in: https://cloud.google.com/container-registry/docs/advanced-authentication
Error: gocdk launch: cloud run launch: docker push: exit status 1
```

The referenced page told me to run `gcloud auth configure-docker`; after I did that it worked (or at least got past this step).

Should we run this command for the user?

**Comment by vangent** [comment]
The next error (feel free to split into a separate bug):

```
Error: gocdk launch: cloud run launch: googleapi: Error 403: Cloud Run API has not been used in project XXX before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/run.googleapis.com/overview?project=892942638129 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry., accessNotConfigured
```

**Comment by vangent** [comment]
That error persisted over ~10 minutes; I had to go to the referenced web page and enable the Cloud Run API.

**Comment by vangent** [comment]
Next error:

```
Error: gocdk launch: cloud run launch: googleapi: Error 404: Requested entity was not found., notFound
```

**Comment by vangent** [comment]
That turned out to be because I had set `location` to `us-west`, and apparently CloudRun is singly-homed in `us-central1` for now.

**Comment by vangent** [comment]
After that it worked!

**Comment by zombiezen** [comment]
Notes:

1. [This TODO](https://github.com/google/go-cloud/blob/f82c134f9fa58dd90310efb479f341bf25ecf419/internal/cmd/gocdk/internal/launcher/cloudrun.go#L54) still needs to be fixed so that the user knows to configure `gcloud` to auth to Docker.
2. The Cloud Run API enablement was fixed in #2545.
3. The us-central1 element has been fixed.

The first item is the only element that's remaining.
